src/layers/core/arc-layer/arc-layer.js

```diff
@@ -41,15 +41,20 @@ export default class ArcLayer extends Layer {
 
     const {attributeManager} = this.state;
     attributeManager.addInstanced({
-      instancePositions: {size: 4, update: this.calculateInstancePositions},
+      instancePositions: {
```

Comment:
Accessor is assumed to be a string across `AttributeManager`. If we allow it to be an array more inspection and testing are needed. Maybe in a separate diff?